### PR TITLE
Tutorial: Add note about redirection

### PR DESCRIPTION
**Pull request by SanketDG**
Fixes https://github.com/xonsh/xonsh/issues/1863

Let me if I need to add anything!

### CODE REVIEW

**Comment by gforsyth** [comment]
Hey @SanketDG, thanks so much for putting this in!  

I think we need to be a little more specific about whitespace and redirection given that we can redirect streams with words and those words should not be separated by whitespace.  Maybe tweak the note to indicate that the target of the redirection should be separated by a space?  

Also, could you add an entry in the `news` folder?  Instructions are available here: https://xonsh.github.io/xonsh-docs/dev/devguide.html#changelog

Thanks!

**Comment by scopatz** [comment]
Thanks for the PR @SanketDG!

**Comment by gforsyth** [comment]
Hey @SanketDG -- any interest in continuing work on this?  If not I can take it on but we'd be happy to have your contribution!

**Comment by SanketDG** [comment]
Sorry, this fell through. I have updated my PR.

File: news/fix-input-output-docs.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+**Added:** * Add note that the target of redirection should be separated by a
```
Comment (gforsyth) [inline news/fix-input-output-docs.rst]: `**Added:**` should be on it's on line, then an empty line, then the bullet point

**Comment by gforsyth** [comment]
Hey @SanketDG -- awesome!  Thanks so much for fixing this up.  There's just one little hiccup in the news entry that the linter is upset about.

**Comment by SanketDG** [comment]
@gforsyth Done thanks!

**Comment by gforsyth** [comment]
Thanks a ton, @SanketDG !

**Pull request merged**